Source/WTF/wtf/Assertions.cpp

```diff
@@ -326,7 +326,7 @@ void WTFCrash()
 #else
     *(int *)(uintptr_t)0xbbadbeef = 0;
     // More reliable, but doesn't say BBADBEEF.
-#if COMPILER(GCC_COMPATIBLE)
+#if COMPILER(GCC_COMPATIBLE) || COMPILER(CLANG)
```

Comment:
I think this is really confusing, since one expects that GCC_COMPATIBLE to mean "GCC or clang". 

Apparently clang-cl doesn't define `__GNUC__`? I think that means we should do one of the following:
1. Update the definition of GCC_COMPATIBLE (unless this is unsafe for some separate reason)
2. Drop _COMPATIBLE and explicitly say `COMPILER(GCC) || COMPILER(CLANG)` (after all, GCC is defined as "GCC-compatible but not clang")
3. Add a new define and say `COMPILER(GCC_COMPATIBLE) || COMPILER(CLANG_CL)`

Reply:
#2 seems good. Will fix.

Reply:
Fixed.